.circleci/config.yml

```diff
@@ -118,7 +118,18 @@ jobs:
             - v1-dependencies-{{ checksum "pom.xml" }}
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
-      - run: mvn -s .circleci/.circleci.settings.xml clean install
+      - when:
```

Comment:
It will not work with this config, we need a "publish" job (see https://github.com/Jahia/sandbox/blob/main/.circleci/config.yml#L163 and https://github.com/Jahia/sandbox/blob/main/.circleci/config.yml#L604)